pennylane_lightning/core/src/simulators/lightning_tensor/cutn/base/tests/Tests_tensornetBase.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2024 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the License);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an AS IS BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <complex>
+#include <vector>
+
+#include <catch2/catch.hpp>
+
+#include "TypeList.hpp"
+
+#include "TestHelpersMPSCutn.hpp"
+
+using namespace Pennylane::LightningTensor::Cutn::Util;
+
+/**
+ * @file
+ *  Tests for functionality defined in the MPSBase class.
+ */
+
+template <typename TypeList> void testTensornetBase() {
+    if constexpr (!std::is_same_v<TypeList, void>) {
+        using MPST = typename TypeList::Type;
```

Comment:
```suggestion
        using MPS_T = typename TypeList::Type;
```

We can use `_T` for types from templated parameters.